src/Utils.hs

```diff
@@ -134,17 +134,22 @@ attribs = ["shape", "color", "label", "scale", "position"]
 attribVs = shapes
 shapes =  ["Auto", "None", "Circle", "Box", "SolidArrow", "SolidDot", "HollowDot", "Cross"]
 labelrws = ["Label", "AutoLabel", "NoLabel"]
-dsll = ["tconstructor","vconstructor","operator","forvars","fortypes","predicate", "Prop", "type"]
+dsll = ["tconstructor","vconstructor","operator","forvars","fortypes","predicate", "Prop", "type", "<:", ":=", "<->"]
```

Comment:
we don't have `:=` in the DSLL; it got replaced by "->".
e.g. `notation "f.domain" -> "Dom(f)"`